### PR TITLE
refactor, chore(egen): Removes boilerplate, heading fixes, and other corrections from template

### PR DESCRIPTION
**Pull request by nileshspringml**
**REQUIRED:** 
- Updated template of notebook based on new template
- Tested code end to end and its working fine

<br>
Notebook template is updated and tested notebook end to end
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 1]
```diff
@@ -32,25 +33,27 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/prediction/get_started_with_raw_predict.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fprediction%2Fget_started_with_raw_predict.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_raw_predict.ipynb:43]: update colab enterprise logo URL:

- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 2]
```diff
@@ -127,186 +130,122 @@
...
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "install_aip"
-      },
-      "outputs": [],
-      "source": [
-        "! pip3 install --upgrade --quiet google-cloud-aiplatform \n",
-        "! pip3 install tensorflow-hub"
+        "## Getting Started"
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_raw_predict.ipynb:136]: Get started

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 3]
```diff
@@ -127,186 +130,122 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "58707a750154"
+        "id": "aae9ca040eab"
       },
       "source": [
-        "### Colab only: Uncomment the following cell to restart the kernel."
+        "### Install Vertex AI SDK and other required packages\n"
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_raw_predict.ipynb:145]: add "for Python":

- Vertex AI SDK for Python

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 4]
```diff
@@ -127,186 +130,122 @@
...
+      "cell_type": "markdown",
       "metadata": {
-        "id": "region"
+        "id": "54c5ef8a8f43"
       },
-      "outputs": [],
       "source": [
-        "REGION = \"us-central1\"  # @param {type: \"string\"}"
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_raw_predict.ipynb:196]: remove "Please" and contract "it is":

- Wait until it's finished...

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 5]
```diff
@@ -565,13 +481,13 @@
       "source": [
         "## Creating an `Endpoint` resource\n",
         "\n",
-        "You create an `Endpoint` resource using the `Endpoint.create()` method. At a minimum, you specify the display name for the endpoint. Optionally, you can specify the project and location (region); otherwise the settings are inherited by the values you set when you initialized the Vertex AI SDK with the `init()` method.\n",
+        "You create an `Endpoint` resource using the `Endpoint.create()` method. At a minimum, you specify the display name for the endpoint. Optionally, you can specify the project and location; otherwise the settings are inherited by the values you set when you initialized the Vertex AI SDK with the `init()` method.\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_raw_predict.ipynb:484]: Vertex AI SDK for Python

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb (hunk 3 above)
Comment (nileshspringml) [reply notebooks/official/prediction/get_started_with_raw_predict.ipynb]: Fixed the issue

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 6]
```diff
@@ -127,186 +130,122 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "gcp_authenticate"
+        "id": "107c51893a64"
       },
       "source": [
-        "**3. Colab, uncomment and run:**"
+        "### Set Google Cloud project information and initialize Vertex AI SDK\n",
```
[the remark this answers is not in the document]
Comment (nileshspringml) [reply notebooks/official/prediction/get_started_with_raw_predict.ipynb]: Fixed the issue

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb (hunk 4 above)
Comment (nileshspringml) [reply notebooks/official/prediction/get_started_with_raw_predict.ipynb]: Fixed the issue

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb (hunk 2 above)
Comment (nileshspringml) [reply notebooks/official/prediction/get_started_with_raw_predict.ipynb]: Fixed the issue

**Comment by katiemn** [comment]
Is this draft pull request ready for review?

**Comment by nileshspringml** [comment]
> Is this draft pull request ready for review?

Yes, ready for PR review

File: notebooks/official/prediction/get_started_with_raw_predict.ipynb [hunk 7]
```diff
@@ -8,6 +8,7 @@
       },
       "outputs": [],
       "source": [
+        "# @title Copyright & License (click to expand)\n",
```
Comment (katiemn) [inline notebooks/official/prediction/get_started_with_raw_predict.ipynb:11]: Can you please remove this line?

Comment (nileshspringml) [reply notebooks/official/prediction/get_started_with_raw_predict.ipynb]: We removed required line

**Review by katiemn — APPROVED** [review]

**Pull request merged**